content/docs/1.6.0/references/helm-values.md

```diff
@@ -182,8 +182,8 @@ Please also refer to this document [ocp-readme](https://github.com/longhorn/long
 | annotations | `{}` | Annotations to add to the Longhorn Manager DaemonSet Pods. Optional. |
 | enableGoCoverDir | `false` | Enable this to allow Longhorn to generate code coverage profiles |
 | enablePSP | `false` | For Kubernetes < v1.25, if your cluster enables Pod Security Policy admission controller, set this to `true` to ship longhorn-psp which allow privileged Longhorn pods to start |
-| helmPreUpgradeCheckerJob.enabled | `true` |  |
-| metrics.serviceMonitor.enabled | `false` |  |
+| helmPreUpgradeCheckerJob.enabled | `true` | Enable this to allow Longhorn to do pre-upgrade checks. Disable this to support ArgoCD. |
```

Comment:
```suggestion
| helmPreUpgradeCheckerJob.enabled | `true` | Setting that allows Longhorn to perform pre-upgrade checks. Disable this setting when installing Longhorn using Argo CD. |
```

Reply:
Note: The official spelling of [Argo CD](https://argoproj.github.io/cd/) has a space.

Reply:
>Setting that allows Longhorn to perform pre-upgrade checks. Disable this setting when installing Longhorn using Argo CD or other GitOps solutions. 

@mantissahz @jillian-maroket

Reply:
Modified. Thanks.